tests/logger/test_logger.py

```diff
@@ -166,6 +169,70 @@ def test_set_signal(self, run, tick, signal_id, state_before, state_after):
         assert log_entry.state_before == state_before
         assert log_entry.state_after == state_after
 
+    @freeze_time()
+    def test_train_enter_block_section(
+        self, run, tick, train_id, block_section_id, block_section_length
+    ):
+        logger = Logger(run_id=run.id)
+        logger.train_enter_block_section(
+            tick=tick,
+            train_id=train_id,
+            block_section_id=block_section_id,
+            block_section_length=block_section_length,
+        )
+        log_entry = (
+            TrainEnterBlockSectionLogEntry.select()
+            .where(
+                TrainEnterBlockSectionLogEntry.tick == tick
+                and TrainEnterBlockSectionLogEntry.train_id == train_id
+                and TrainEnterBlockSectionLogEntry.block_section_id == block_section_id
+            )
+            .first()
+        )
+        assert log_entry.timestamp == datetime.now()
+        assert log_entry.tick == tick
```

Comment:
This line seems very redundant, because you already only selected those entries from the database.

Reply:
Yes also thought about this. but for completness sakes I just left it how it is. _Frei nach dem Motto: Doppelt hält besser_